diff --git a/src/video.c b/src/video.c
index fd4dd7e..36f7990 100644
--- a/src/video.c
+++ b/src/video.c
@@ -17,6 +17,8 @@
 #include "sdcard.h"
 #include "i2c.h"
 
+#include "cpu/fake6502.h"
+
 #include <limits.h>
 #include <stdint.h>
 
@@ -1306,7 +1308,7 @@ uint8_t video_read(uint8_t reg, bool debugOn) {
                        io_rddata[reg - 3] = video_space_read(io_addr[reg - 3]);
 
                        if (log_video) {
-                               printf("READ  video_space[$%X] = $%02X\n", address, value);
+                               printf("# READ  video_space[$%X] = $%02X\n", address, value);
                        }
                        return value;
                }
@@ -1351,6 +1353,10 @@ void video_write(uint8_t reg, uint8_t value) {
        //      printf("ioregisters[0x%02X] = 0x%02X\n", reg, value);
        // }
        //      printf("ioregisters[%d] = $%02X\n", reg, value);
+       if (log_video) {
+               printf("%09u, 0x%02x, 0x%02x\n", clockticks6502, reg & 0x1F, value);
+       }
+
        switch (reg & 0x1F) {
                case 0x00:
                        io_addr[io_addrsel] = (io_addr[io_addrsel] & 0x1ff00) | value;
@@ -1369,7 +1375,7 @@ void video_write(uint8_t reg, uint8_t value) {
                case 0x04: {
                        uint32_t address = get_and_inc_address(reg - 3);
                        if (log_video) {
-                               printf("WRITE video_space[$%X] = $%02X\n", address, value);
+                               printf("# WRITE video_space[$%X] = $%02X\n", address, value);
                        }
                        video_space_write(address, value);
 
